testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -2924,6 +2924,114 @@ public void testSparkAlterStructColumnType(StorageFormat storageFormat)
         onSpark().executeQuery("DROP TABLE " + sparkTableName);
     }
 
+    @Test(groups = {ICEBERG, PROFILE_SPECIFIC_TESTS})
+    public void testSparkReadingTrinoBloomFilters()
+    {
+        String baseTableName = "test_spark_reading_trino_bloom_filters";
+        String trinoTableName = trinoTableName(baseTableName);
```

Comment:
you  should use `parquet_use_bloom_filter`  set to `true` in the queries right?

Reply:
This property defaults to true?

Reply:
This property default to `false`

https://github.com/trinodb/trino/blob/6962ad23c3a009da25bf703b8258dd8628f69822/plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergSessionProperties.java#L131-L135

Reply:
That's ORC, in parquet it should be enabled by default already